### DISCUSSION TITLE
Likelihood based scoring system

### DISCUSSION THREAD

**Discussion by OVVO-Financial** [microprediction/microprediction#75] [General]
Have you investigated the benefits of a likelihood based scoring system vs. the expanding window currently in place, especially regarding tail observations, which have **immense** implications in daily prizes due to the small number of observations for some of the longer frequency streams.

**Comment by microprediction** [comment]
I'm waiting on a proposal which should contain:

1.  Proposed scoring rule ... *presumably* involving the promotion of 225 samples into a density by some convention?
2. What a player with view P should do ... if that is not obvious ... should they perform an optimization? 
3. How the results of the game should be interpreted ... if the answer is not "just use the best one" or "use an in-house meta-model".  

I will say in advance of a different answer than (3) that there is one likely drawback:

 - A likelihood contest does not really *combine* the participants' forecasts the way a market does, and this might be a problem if, the collective is better than the best entrant (people filling in holes etc). 

Short version of my knee-jerk response: "markets > contests". The options market is more powerful than a contest, it would [seem](https://medium.com/geekculture/the-options-market-beat-94-of-participants-in-the-m6-financial-forecasting-contest-fa4f47f57d33).  However I think there is a sneaky hybrid approach.
